pkg/apis/application/v1alpha1/types.go

```diff
@@ -641,6 +641,8 @@ type SyncOperationResource struct {
 	Kind      string `json:"kind" protobuf:"bytes,2,opt,name=kind"`
 	Name      string `json:"name" protobuf:"bytes,3,opt,name=name"`
 	Namespace string `json:"namespace,omitempty" protobuf:"bytes,4,opt,name=namespace"`
+	// nolint:govet
+	Exclude bool `json:"-"`
```

Comment:
Why `json:"-"` instead of similar to the above fields?

Reply:
Since this filed is only used by the CLI currently, I thought it can be ignored by the json encoder package. Let me know if I have to add the tags on this one too.

Reply:
Ah for sure. Sounds good!